src/seamless_communication/models/inference/translator.py

```diff
@@ -56,11 +56,11 @@ def __init__(
         sample_rate: int = 16000,
     ):
         super().__init__()
+        dtype = torch.float16 if "cuda" in device.type else torch.float32
```

Comment:
nit: what if we also add dtype as the constructor argument?

Reply:
And set this in scripts/m4t/predict.py? This sounds great!